common/auth/tls.go

```diff
@@ -25,6 +25,11 @@ type (
 	TLS struct {
 		Enabled bool `yaml:"enabled"`
 
+		// for Postgres: see more https://www.postgresql.org/docs/9.1/libpq-ssl.html
+		// default to require if Enable is true
+		// TODO: support MySQL/Cassandra as well
```

Comment:
@JosefWN thanks for providing the idea. Do you happen to know how to support different modes for MySQL/Cassandra as well?

Reply:
Wow, that was quick, thanks! I've been using Postgres for the past few years, but MySQL seems quite similar: https://dev.mysql.com/doc/refman/8.0/en/connection-options.html#option_general_ssl-mode

Postgres, for reference: https://www.postgresql.org/docs/13/libpq-ssl.html#LIBPQ-SSL-SSLMODE-STATEMENTS

Not sure about Cassandra, perhaps more info here: https://cassandra.apache.org/doc/latest/operating/security.html